tests/integration/targets/latest/tasks/netbox_power_panel.yml

```diff
@@ -0,0 +1,85 @@
+---
+# © 2020 Nokia
+# Licensed under the GNU General Public License v3.0 only
+# SPDX-License-Identifier: GPL-3.0-only
+##
+##
+### NETBOX_POWER_PANEL
+##
+##
+- name: "POWER_PANEL 1: Necessary info creation"
+  netbox.netbox.netbox_power_panel:
+    netbox_url: http://localhost:32768
+    netbox_token: 0123456789abcdef0123456789abcdef01234567
+    data:
+      name: Power Panel
+      site: Test Site
+    state: present
+  register: test_one
+
+- name: "POWER_PANEL 1: ASSERT - Necessary info creation"
+  assert:
+    that:
+      - test_one is changed
+      - test_one['diff']['before']['state'] == "absent"
+      - test_one['diff']['after']['state'] == "present"
+      - test_one['power_panel']['name'] == "Power Panel"
+      - test_one['power_panel']['site'] == 1
+      - test_one['msg'] == "power_panel Power Panel created"
+
+- name: "POWER_PANEL 2: Create duplicate"
+  netbox.netbox.netbox_power_panel:
+    netbox_url: http://localhost:32768
+    netbox_token: 0123456789abcdef0123456789abcdef01234567
+    data:
+      name: Power Panel
+      site: Test Site
+    state: present
+  register: test_two
+
+- name: "POWER_PANEL 2: ASSERT - Create duplicate"
+  assert:
+    that:
```

Comment:
Do you minding adding an update test to add to a rackgroup?

Reply:
I totally forgot this test but interestingly it doesn't work anyway.
The `power-panel` endpoint is expecting a `rack_group` but the `rack_group` parameter is rewritten to `group` in `netbox_utils.py`.

Do you have any hint how I can disable the conversion from `rack_group` to `group` in this case?

Reply:
It looks like your update resolved this. I would have handled it the same way at this point, but I'll look further into it.